react/AppLinker/index.jsx

```diff
@@ -2,40 +2,20 @@ import PropTypes from 'prop-types'
 import React from 'react'
 
 import { withClient } from 'cozy-client'
-import {
-  checkApp,
-  startApp,
-  isMobileApp,
-  isMobile,
-  openDeeplinkOrRedirect,
-  isAndroid,
-  isFlagshipApp
-} from 'cozy-device-helper'
+import { isFlagshipApp } from 'cozy-device-helper'
 import { WebviewContext } from 'cozy-intent'
 import logger from 'cozy-logger'
 
-import expiringMemoize from './expiringMemoize'
 import {
   generateUniversalLink,
   generateWebLink,
   getUniversalLinkDomain
 } from './native'
-import { NATIVE_APP_INFOS } from './native.config'
-
-const expirationDelay = 10 * 1000
-const memoizedCheckApp = expiringMemoize(
-  appInfo => checkApp(appInfo).catch(() => false),
-  expirationDelay,
-  appInfo => appInfo.appId
-)
 
 export class AppLinker extends React.Component {
   static contextType = WebviewContext
 
-  state = {
-    nativeAppIsAvailable: null,
-    isFetchingAppInfo: false
-  }
+  state = {}
```

Comment:
est-ce qu'on a réellement besoin d'un tel state ? 🤔

Reply:
Oui parce que plus loin on utilise `this.setState`

Reply:
on pourrait peut-être mettre state = {{ imgRef: null }} ou quelque chose comme ça ? Ca apporte une meilleure visibilité je trouve.

Reply:
Bien vu